qpmodel/ClassDiagram1.cd

```diff
@@ -0,0 +1,1825 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
```

Comment:
What is this file?

Reply:
This is a UML generated by VS , I use it to understand the code,  I will use. gitignore to delete it.